compiler/rustc_target/src/spec/teeos_base.rs

```diff
@@ -0,0 +1,35 @@
+use super::{Cc, LinkerFlavor, Lld, PanicStrategy};
+use crate::spec::{RelroLevel, TargetOptions};
+
+pub fn opts() -> TargetOptions {
+    let lld_args =
+        &["-zmax-page-size=4096".into(), "-znow".into(), "-ztext".into(), "--execute-only".into()];
```

Comment:
```suggestion
        &["-zmax-page-size=4096", "-znow", "-ztext", "--execute-only"];
```
`into`s are no longer necessary here and in `cc_args`.